poster/styles/styles.scss

```diff
@@ -182,18 +293,56 @@ ul{
   list-style-type: none;
   text-align: center;
   text-transform: uppercase;
+
+  @media only screen and (max-width: 700px){
+    grid-column:2/7;
+    grid-row: 6/7;
+  }
+
+  @media only screen and (max-width: 500px){
+    grid-column:1/9;
+    grid-row: 6/7;
+  }
+
+
 }
 
+// smaller extra info
+
 .date-time{
-  grid-column: 2/4;
+  align-self: center;
+  grid-column: 1/5;
   grid-row: 2/3;
   letter-spacing: 0.1rem;
   text-align: center;
+
+  @media only screen and (max-width: 700px){
+    grid-column: 7/8;
+  }
+
+}
+
+.caption__madegoods{
+  grid-column: 2/4;
+  grid-row: 1/2;
+  text-align: center;
+
+  @media only screen and (max-width: 700px){
+    grid-column: 1/7;
+    grid-row: 2/3;
+    margin: 2rem;
+  }
 }
 
+// social links
+
 .link__socials{
```

Comment:
Did you think about laying these out horizontally or centered on mobile?
<img width="360" alt="mobile-links" src="https://user-images.githubusercontent.com/2653980/47398824-e1533300-d703-11e8-8cb1-b24d69c3f8fb.png">